blueprints/emberfire/index.js

```diff
@@ -8,7 +10,24 @@ module.exports = {
     // to us
   },
 
+  availableOptions: [
+    { name: 'url', type: String }
+  ],
+
+  locals: function(options) {
+    if (!options.url) {
+      throw new SilentError('you need to provide a firebase url e.g. ember generate emberfire --url=https://YOUR-FIREBASE-NAME.firebaseio.com/')
```

Comment:
I'm throwing an error here to let the user know they need to provide a url otherwise we won't know what to put into `config/environments`. Not sure if it's preferable to just put in a placeholder to be replaced later.